helm/prometheus-meta-operator/templates/rbac.yaml

```diff
@@ -93,6 +93,16 @@ rules:
       - create
       - delete
       - update
+  - apiGroups:
+      - cilium.io
+    resources:
+      - ciliumnetworkolicies
```

Comment:
```suggestion
      - ciliumnetworkpolicies
```
you have a typo ;)

Reply:
Oh wow  thx ! I would have definitely missed it !